$OpenBSD$

with_internal_maximum_log_level :
  replace \+ by + as it should be
  add -E option to grep, handle the + in expression

Maybe we should use grep '^[0-9]$', log level > 9 ?

--- configure.orig	Mon Oct 31 17:05:45 2011
+++ configure	Tue Nov  1 14:43:34 2011
@@ -13240,7 +13240,7 @@ with_max_log_level="<unset>"
 if test "${with_internal_maximum_log_level+set}" = set; then :
   withval=$with_internal_maximum_log_level;
     if test "x${withval}" != "xno"; then
-       if echo "${withval}" | grep '^[0-9]\+$' >/dev/null 2>/dev/null; then
+       if echo "${withval}" | grep -E '^[0-9]+$' >/dev/null 2>/dev/null; then
           { $as_echo "$as_me:${as_lineno-$LINENO}: ignoring any EINA_LOG() with level greater than ${withval}" >&5
 $as_echo "$as_me: ignoring any EINA_LOG() with level greater than ${withval}" >&6;}
 
